sites/hashai/resources/docs/simulation/tutorials/hello-hash.md

```diff
@@ -38,69 +38,40 @@ Beneath the workspace's right-hand view-pane, you'll find the simulation control
 
 However, if you toggle back from the raw output view to the 3D viewer, you may notice... nothing at all. Alice and Bob haven't been rendered, because they haven't been given a position in space. Let's go ahead and fix that.
 
-```javascript
+```json
 [
   {
-    agent_name: "Alice",
-    position: [0, 0],
+    "agent_name": "Alice",
+    "position": [0, 0]
   },
   {
-    agent_name: "Bob",
-    position: [2, 0],
-  },
-];
+    "agent_name": "Bob",
+    "position": [2, 0]
+  }
+]
 ```
 
 When you've finished adding positions to your agents, reset the simulation. You should now see two blocks in the 3D viewer representing our agents.
 
 ### Saying Hello
 
-Alice and Bob aren't very interesting right now. Let's teach them some manners. We can give the agents [behaviors](/docs/simulation/creating-simulations/behaviors) that enable them to act and respond to each other. In **init.json** let's add some file names into each of the behavior arrays.
+Alice and Bob aren't very interesting right now. Let's teach them some manners. We can give the agents [behaviors](/docs/simulation/creating-simulations/behaviors) that enable them to act and respond to each other. In `init.json` let's add some file names into each of the behavior arrays.
 
-<Hint style="info">
-You can build Python behaviors instead of JavaScript behaviors if you prefer. Just make sure to name your files with a ".py" ending.
-</Hint>
```

Comment:
I'd keep this hint, it's not talking about `init.py` but actually about the behaviours themselves. We then give examples of Python behaviours below so I think we should have this

Reply:
Good catch, re-added in 5d05ed9714cac0adf4a0107374992787ef0f7259